dockerfile-latest

```diff
@@ -0,0 +1,41 @@
+FROM airsharedcontainers.azurecr.io/vstsdropdownloadcore:master AS app
+FROM airsharedcontainers.azurecr.io/msftkube AS msftkube
+FROM microsoft/vsts-agent:ubuntu-16.04-docker-17.12.0-ce-standard
+
+# Get vstsdropdownloadcore's built drop downloader application
+copy --from=app /app/* /app/
+copy --from=msftkube /msftkube/src/dist/*.wh /msftkube/package.wh
```

Comment:
are we copying just msftkube or all it's dependencies?

Reply:
This will copy msftkube's wheel and should pull in it's dependencies when installed.